docs/getting-started/quickstart.md

```diff
@@ -61,31 +63,26 @@ Run the following command to deploy your app to DBOS Cloud:
 npx dbos-cloud app deploy
 ```
 
-This command first prompts you to login/sign up for DBOS Cloud.
-
-Then, it prompts you to provision a free [database instance](../cloud-tutorials/database-management.md) for your application.
-
-Finally, it uploads your code to DBOS Cloud and deploys your application.
-After about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
+This command first prompts you to login, or register if this is your first time. Then, it prompts you to provision a free cloud database instance. Finally, it uploads your code to DBOS Cloud and deploys your app. After about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
```

Comment:
```suggestion
This command first prompts you to login, or register if this is your first time. Then, it prompts you to provision a free cloud database instance. Finally, it uploads your code to DBOS Cloud and deploys your app. In less than a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
```

Reply:
Unlike promotional materials, I think doc can set medium expectations and the software can then pleasantly exceed them? See what you think about my latest?

Reply:
Should be all set now